plugins/woocommerce/includes/class-wc-ajax.php

```diff
@@ -619,7 +621,13 @@ public static function add_attribute() {
 		$attribute->set_name( sanitize_text_field( wp_unslash( $_POST['taxonomy'] ) ) );
 		/* phpcs:disable WooCommerce.Commenting.CommentHooks.MissingHookComment */
 		$attribute->set_visible( apply_filters( 'woocommerce_attribute_default_visibility', 1 ) );
-		$attribute->set_variation( apply_filters( 'woocommerce_attribute_default_is_variation', 1 ) );
+		$attribute->set_variation(
+			apply_filters(
```

Comment:
Would reverting [to the previous functionality ](https://github.com/woocommerce/woocommerce/pull/38003)be sufficient?

Reply:
This would also fix the test-- setting a simple product as the default could create more edge cases in the future 🤔

Reply:
@AnnaMag I don't see any failing test. What test are you referring to?

I'd prefer not to revert to the previous functionality, since Jarek has been quite set that this change will help prevent confusion based on his user research.